index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Portfolio Website</title>
+
+ <!--icon link-->
+ <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css">
+ <script src="https://kit.fontawesome.com/9ea65bb5f9.js" crossorigin="anonymous"></script>
+
+ <!--css link-->
+ <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+  <div class="wrapper">
+    <!--header section-->
+    <header class="header">
+
+      <!--logo-->
+      <a href="#index.html" class="logo">My logo</a>
+
+      <!--navbar-->
+      <nav>
+        <ul class="navlink">
+          <li><a href="#index.html">Portfolio</a></li>
+          <li><a href="#about.html">About</a></li>
+          <li><a href="#contact.html">Contact</a></li>
+        </ul>
+      </nav>
+      <!--end of navbar-->
+
+      <div class="toggle">
+        <i class="uil uil-bars"></i>
```

Comment:
- [ ] I would suggest you please use the same color and font defined in the template.
### Your design
![hisssssss](https://user-images.githubusercontent.com/81727988/191942430-70abc9d6-3b65-4ee6-9d68-6bcda43c1765.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191942462-78ec141f-068d-496e-80cb-82299742d602.png)